ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -2664,4 +2664,12 @@ msgstr "Aller à la page suivante"
 
 #: ckanext/ontario_theme/page.py:51
 msgid "Go to page {}"
-msgstr "Aller à page {}"
\ No newline at end of file
+msgstr "Aller à page {}"
```

Comment:
where am I supposed to see this?

Reply:
This is a merge conflict resolution, it was implemented with the pagination fixes.

Reply:
Is it accessibility text? I don't see this text with the pagination buttons

Reply:
<img width="284" alt="image" src="https://github.com/ongov/ckanext-ontario_theme/assets/60950452/2df3df8b-6489-48c7-b86f-d79204105e7a">

Yes, but it is already on main. This was just a merge conflict and this line wasn't actually changed, there was just a newline added after.